Mars-QA/SpecFlow/Feature.feature

```diff
@@ -0,0 +1,78 @@
+﻿
+Feature: Feature
+
+
+As a User Create a Profile then add languages, skills so that recuriter see your skills and language
```

Comment:
can you please separate your skills and language tests for example you should have a feature file for skills and a feature file for language